security/file_integrity_operator/file-integrity-operator-release-notes.adoc

```diff
@@ -15,6 +15,24 @@ For an overview of the File Integrity Operator, see xref:../../security/file_int
 
 To access the latest release, see xref:../../security/file_integrity_operator/file-integrity-operator-updating.adoc#olm-preparing-upgrade_file-integrity-operator-updating[Updating the File Integrity Operator].
 
+[id="file-integrity-operator-release-notes-1-3-6"]
+== OpenShift File Integrity Operator 1.3.6
+ 
+The following advisory is available for the OpenShift File Integrity Operator 1.3.6:
+
+* link:https://access.redhat.com/errata/RH??-2025:????[RH??-2025:???? OpenShift File Integrity Operator Bug Fix Update]
+
+[id="file-integrity-operator-1-3-6-bug-fixes"]
+=== Bug fixes
+
+* Previously, when FIO declared `Failure`, the user could not clear the state and re-test. With this release, FIO allows the user to re-initialize after `Failure` when the user asks FIO to clear the state of the scan, allowing the user to re-run the scan without retaining the prior `Failure` status. (link:https://issues.redhat.com/browse/OCPBUGS-18933[*OCPBUGS-18933*])
+
+* Previously, after a reset of FIO, the metric file_integrity_operator_node_failed would be reset, which would cause an existing alert `NodeHasIntegrityFailure` to disappear. With this release, `NodeHasIntegrityFailure` is preserved after an FIO restart. (link:https://issues.redhat.com/browse/OCPBUGS-42807[*OCPBUGS-42807*])
+
+* Previously, when a new node is added to a cluster by scaling up the `machineset`, FIO would mark the newly added node as `Failed` before the node is ready. With this release FIO waits for the newly added node to complete installation. (link:https://issues.redhat.com/browse/OCPBUGS-36483[*OCPBUGS-36483*])
+
+* Previously, the `aide-openshift-file-integrity-daemon` pod would constantly force-initialize the AIDE database. With this release, FIO initializes the AIDE database only once. (link:https://issues.redhat.com/browse/OCPBUGS-37300[*OCPBUGS-37300*])
```

Comment:
I assume it is obvious to your readers what "AIDE" stands for as an abbreviation/acronym. If not, then you might want to consider adding the unabbreviated name here.